### PR TITLE
Setting up Privacy Policy

### LINKED ISSUE #6: FEATURE: Privacy Policy
# User Story

* As a user
* I can see a privacy policy
* So that I know how my data will be used

# Discussion

It is important for all systems that process sensitive information to disclose how that information will be used.

# Acceptance Criteria

- [x] There is a new Page component called <PrivacyPage>
- [x] This page component has placeholder text:  `<h2>Privacy Policy</h2>`
- [x] Beneath the `h2` tag, the custom HTML below should be included to publish a privacy policy from termly.io
- [x] This page is available at the link `/privacy` in the application (this is done in `App.js`)
- [x] The `<Footer>` component has the text `Privacy Policy` somewhere suitable, with a link to `/privacy`

# Custom HTML

Custom HTML is in the file: 

https://github.com/ucsb-cs156/proj-gauchoride/blob/main/frontend/public/privacy.html

This may be copied directly into the PrivacyPage component.

### PR DESCRIPTION
**Pull request by Danny-You**
As a user
I can see a privacy policy
So that I know how my data will be used

Closes #6 

image

### CODE REVIEW

**Review by lawrencewang30 — DISMISSED** [review]
Code and screenshot looks good to me, can merge this branch after others send code review

**Review by pconrad — DISMISSED** [review]
The issue asked for the privacy policy link to be in the footer, not a new button in the header.

The location next to the Login button is kind of distracting.

Can we remove that, and instead put the link in the footer where the issue requested it?

**Comment by Danny-You** [comment]
Addressed the issue, removed the button for Privacy Policy in the header, added the link in the footer where the issue requested it. Please give it another review.

image

**Review by Kanagawa-okiNamiUra — APPROVED** [review]
Checked the running instance at Danny's localhost; looks good to me

**Review by InariHimeko — APPROVED** [review]
All the code LGTM

**Review by GDW1 — APPROVED** [review]
This PR seems good to me

File: frontend/src/main/components/Nav/Footer.js [hunk 1]
```diff
@@ -44,6 +44,9 @@ export default function Footer() {
           <a data-testid="footer-sticker-link" href="https://www.as.ucsb.edu/sticker-packs">www.as.ucsb.edu/sticker-packs/</a>
           .
         </p>
+        <p>
+          <a href="/privacy">Privacy Policy</a>
```
Comment (chriscerie) [inline frontend/src/main/components/Nav/Footer.js:48]: Use react-router’s `Link` over anchor tags to avoid full page reloads.

**Review by chriscerie — CHANGES_REQUESTED** [review]

**Comment by pconrad** [comment]
I'm going to approve this, but I'm also going to ask you to fix this in another PR.

image

File: frontend/src/main/components/Nav/Footer.js (hunk 1 above)
Comment (pconrad) [reply frontend/src/main/components/Nav/Footer.js:48]: Agreed with Chris.

**Pull request merged**